subsys/mgmt/ec_host_cmd/Kconfig.logging

```diff
@@ -10,3 +10,18 @@ config EC_HOST_CMD_LOG_DBG_BUFFERS
 	  Every command is logged with the debug logging level. Use this config
 	  to decide, if full reqest and response buffers are logged alongside
 	  other command parameters.
+
+config EC_HOST_CMD_SUPPRESSED_NUMBER
+	int "Maximum number of suppressed commands"
+	default 8
+	help
+	  Suppressed commands are not logged on host command reception. Set
+	  the maximum number of the suppressed commands. Set 0 to disable
+	  suppressing commands.
+
+config EC_HOST_CMD_SUPPRESSED_INTERVAL
```

Comment:
Should this have a unit? _MS or something. May even be worth using a coarser one and convert it in the code if you don't need the granularity, this is hard to read.

Reply:
Makes sense :-)